Add instrunctions not to use HTML tags in README.md (#211)
optuna/optunahub-registry: template/README.md

---
-->

> Instruction (you can remove this instruction when you write the `README.md` for your package)

Reviewer: ```suggestion
Instruction (Please remove this instruction after you carefully read)
```

- Please read the [tutorial guide](https://optuna.github.io/optunahub-registry/recipes/001_first.html) to register your feature in OptunaHub. You can find more detailed explanation of the following contents in the tutorial.

> - Looking at [other packages' implementations](https://github.com/optuna/optunahub-registry/tree/main/package) will also help you.

Reviewer: ```suggestion
- Looking at [other packages' implementations](https://github.com/optuna/optunahub-registry/tree/main/package) would also be helpful.
```
